index.html

```diff
@@ -363,8 +363,9 @@ <h3 class="contact-me">Contact me</h3>
     
             <button class="submit">Get In Touch</button>
           </form>
```

Comment:
- [ ] As written by the project requirements, please kindly do the following:
- Add `action` and `method` attribute to your form e.g: 
```
<form action="https://formspree.io/f/{form_id}" method="POST" class="form">
</form>
```

- [ ] I kindly suggest that you add a `maxLenght` HTML attribute with a value of 30 to your name input on line number `347`
- [ ] I kindly suggest that you  a `maxLenght` HTML attribute with a value of 500 to your textarea input on line number `355`